forta-uniswap-event-py/README.md

```diff
@@ -0,0 +1,27 @@
+# Large Uniswap Detector Agent
+
+## Description
+
+This agent detects large uniswap transactions on ETH liquidity pools
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- AE-UNISWAP-LARGESWAP-EVENT
+  - Fired when a swap occurs where the value being traded is over 5 ether
+  - Only triggers when a Deposit or Withdrawal event occurs on the WETH contract via the Uniswap V2
+    Router contract
+
+## Test Data
+
+The agent behaviour can be verified with the following transactions:
```

Comment:
Nit: I believe `behaviour` is the British English spelling.
```suggestion
The agent behavior can be verified with the following transactions:
```

Reply:
yea... thats why i spelled it that way.... :P